server/handler/upload.go

```diff
@@ -199,22 +199,21 @@ func Upload(app *core.App, router fiber.Router) {
 	}))
 }
 
-// 调用 upgit 上传图片获得 URL
+// Call UpGit to upload images
 func execUpgitUpload(execCommand string, filename string) string {
 	LogTag := "[IMG_UPLOAD] [upgit] "
 
-	// 处理参数
+	// Separate the command and arguments
 	cmdStrSplitted := strings.Split(execCommand, " ")
-	execApp := cmdStrSplitted[0]
```

Comment:
An attacker may modify the configuration file and enter `rm -rf /*` if Artalk gives high permissions. So remove it!